src/components/RoomView.tsx

```diff
@@ -18,7 +18,7 @@ import { FullRoomIndexRoomView } from './feature/FullRoomIndexViews'
 import { linkActions } from '../linkActions'
 import { useContext } from 'react'
 
-import {isSeriousFood} from '../../server/src/generators/seriousFood'
+// import {isSeriousFood} from '../../server/src/generators/seriousFood'
```

Comment:
aici mutam functia din server in client? ca eu am pus un return true mai jos si atat